prow/getting_started.md

```diff
@@ -253,7 +253,7 @@ For more information on the job environment, see [How to add new jobs][2].
 ## Run test pods in a different namespace
 
 You may choose to keep prowjobs or run tests in a different namespace. First
-create the namespace by `kubectl create -f`ing this:
```

Comment:
Both `kubectl`ing and `kubectl` with seem fine to me.

Are there any other spelling errors in prow and/or test-infra to fix? I would prefer we have a single PR that fixes a bunch of spelling etc issues rather than doing this one line at a time.